extensions/undertow/deployment/src/main/java/io/quarkus/undertow/deployment/UndertowBuildStep.java

```diff
@@ -248,8 +248,8 @@ public void registerUndertowHandlersConf(BuildProducer<ServletExtensionBuildItem
 
     @BuildStep
     List<IgnoredServletContainerInitializerBuildItem> translateDeprecated(
-            List<BlacklistedServletContainerInitializerBuildItem> old) {
```

Comment:
I am not sure this is correct, I think it exists for compatibility purposes, maybe the method can be removed completely along with the `BlacklistedServletContainerInitializerBuildItem` class

Reply:
I removed it.